src/App.js

```diff
@@ -32,6 +32,81 @@ function App() {
     });
   });
 
+  //checking map hover - 1
+  useEffect(() => {
+  if (!map.current) return;
+    map.current.on('load', () => {
+    map.current.addSource('cbs', {
+    'type': 'geojson',
+    'data': 'https://d2ad6b4ur7yvpq.cloudfront.net/naturalearth-3.3.0/ne_50m_admin_0_countries.geojson'
+    });
+
+    map.current.addLayer({
+      'id': 'cf',         // country-fills
+      'type': 'fill',
+      'source': 'cbs',
+      'layout': {},
+      'paint': {
+      'fill-color': '#BFBAAD',
+      'fill-opacity': 0.1
+      }
+    });
+    
+    map.current.addLayer({
+    'id': 'cb',         // country-borders
+    'type': 'line',
+    'source': 'cbs',
+    'layout': {},
+    'paint': {
+    'line-color': '#BFBAAD',
+    'line-width': 1
+    }
+    });
+
+    map.current.addLayer({
+      "id": "cfh",  // country-fills-hover",
+      "type": "fill",
+      "source": "cbs",
+      "layout": {},
+      "paint": {
+          "fill-color": "#BFBAAD",
+          "fill-opacity": 0.5
+      },
+      "filter": ["==", "name", ""]
+    });
```

Comment:
can we move all this into a seperate function and call it initMap ? 

```suggestion
function initMap () => {
    map.current.on('load', () => {
    map.current.addSource('cbs', {
    'type': 'geojson',
    'data': 'https://d2ad6b4ur7yvpq.cloudfront.net/naturalearth-3.3.0/ne_50m_admin_0_countries.geojson'
    });

    map.current.addLayer({
      'id': 'cf',         // country-fills
      'type': 'fill',
      'source': 'cbs',
      'layout': {},
      'paint': {
      'fill-color': '#BFBAAD',
      'fill-opacity': 0.1
      }
    });
    
    map.current.addLayer({
    'id': 'cb',         // country-borders
    'type': 'line',
    'source': 'cbs',
    'layout': {},
    'paint': {
    'line-color': '#BFBAAD',
    'line-width': 1
    }
    });

    map.current.addLayer({
      "id": "cfh",  // country-fills-hover",
      "type": "fill",
      "source": "cbs",
      "layout": {},
      "paint": {
          "fill-color": "#BFBAAD",
          "fill-opacity": 0.5
      },
      "filter": ["==", "name", ""]
    });
 }
 
 useEffect(() => 
 if (!map.current) return; 
 this.initMap()
 
 )
```